src/components/study-manager.js

```diff
@@ -253,10 +288,48 @@ const StudyCard = ({study, onClick}) => {
                     </Button>
                 </DialogActions>
             </Dialog>
+            <RenameDialog study={study}
+                          openRenameDialog={openRenameDialog}
+                          handleCloseDialog={handleCloseRenameDialog}
+                          handleCancel={handleCancelRename}
+                          handleConfirm={handleRenameStudyConfirmed}/>
         </div>
     );
 };
 
+const RenameDialog = (props) => {
+
+    const [newStudyNameValue, setNewStudyNameValue] = React.useState(props.study.studyName);
+
+    const updateStudyNameValue= (event) => {
+        setNewStudyNameValue(event.target.value);
+        console.debug("newStudyName : " + newStudyNameValue);
+    };
+
+    const handleClick = () => {
+        console.log(newStudyNameValue);
+        props.handleConfirm(newStudyNameValue);
+    };
+
+    return (
+        <Dialog open={props.openRenameDialog} onClose={props.handleCloseDialog} aria-labelledby="dialog-title-rename">
+            <DialogTitle id="dialog-title-rename"><FormattedMessage id="renameStudy"/></DialogTitle>
+            <DialogContent>
+                <InputLabel htmlFor="newStudyName"><FormattedMessage id="renameStudyMsg"/></InputLabel>
+                <TextField id="newStudyName" defaultValue={props.study.studyName} required={true} onChange={updateStudyNameValue}></TextField>
```

Comment:
We should either use value and a state (controlled component) or defautlValue and  a ref (uncontrolled component). Controlled components are the officially recommended solution